### ISSUE TITLE
Tag text color does not pass contrast ratio for a11y

### ISSUE THREAD

**Issue by Angelmmiguel** [vmware/clarity#5104]
## Describe the accessibility issue

The default [tag text color](https://clarity.design/storybook/core/?path=/story/components-tag-getting-started--page#color) when no status is applied is #8C8C8C. Since the text is pretty small and the background color is white, the color contrast ratio is very low. You can see an example in the [WebAim color contrast checker](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=09AAAA).

<img width="203" alt="Screenshot of the Tag component. The text is pretty small and the color is a light gray on top of a white background" src="https://user-images.githubusercontent.com/4056725/94826996-39266880-0408-11eb-987a-ef26950a3e87.png">

For now, I have changed the color to #333333 (`--cds-token-color-neutral-900`).

## WCAG Guideline

https://www.w3.org/TR/WCAG21/#contrast-enhanced

## How to reproduce

Not steps are required, just check the default text color in the Clarity Core documentation: https://clarity.design/storybook/core/?path=/story/components-tag-getting-started--page#color

## Expected behavior

The text color has enough color contrast ratio.

## Versions

It doesn't apply

**Comment by mathisscott** [comment]
Good catch. This needs a look from design. I'm wondering if we should go darker neutral or reuse the border color as the text color (assuming that has enough contrast). also the disabled status needs a design review.

CC: @colinreedmiller + @yuxin-ux

**Comment by colinreedmiller** [comment]
The color in the implementation is incorrect. The component known as *tag* in core is equivalent to the *label* in clarity ng which has a text color of #565656 - the equivalent value using the Core token system is neutral-800 which is slightly darker and is compliant. The implementation needs to change the default text color to neutral-800 - I will check the colored text and add to this issue if any changes are required.

**Comment by mathisscott** [comment]
Thanks, @colinreedmiller 

I can fix that on Monday and have a PR ready then.

**Comment by colinreedmiller** [comment]
Excellent @mathisscott - I checked the colored text on colored backgrounds to be sure they are compliant while this is addressed and all as they should be. The only a11y fix required is as mentioned above - the default gray text color.
